### ISSUE TITLE
Detox plugin generates invalid network security XML

### ISSUE THREAD

**Issue by jesse-savary** [expo/config-plugins#58]
### Summary

Currently the Detox plugin outputs the following for `network_security_config.xml`:
```

    <?xml version="1.0" encoding="utf-8"?>
    <network-security-config>
      
    <domain-config cleartextTrafficPermitted="true">
      <domain includeSubdomains="true">10.0.2.2</domain><domain includeSubdomains="true">localhost</domain>
    </domain-config>
  
    </network-security-config>
  
```

The extra whitespace causes a build error during a merge step as the XML declaration (`<?xml`) must be at the **very beginning** of the file.

### Config Plugin

@config-plugins/detox

### What platform(s) does this occur on?

Android

### Reproducible demo

1. add the Detox config plugin
2. run `expo prebuild -p android --no-install`
3. run `expo run:android`

**Comment by kbrandwijk** [comment]
This issue has already been addressed and released in version 1.2.2. of the config plugin: https://github.com/expo/config-plugins/pull/47. Can you make sure you are on the latest version?

**Comment by jesse-savary** [comment]
Hey -- something was cached somewhere. Thank you for the rapid response.
